source/dub/dub.d

```diff
@@ -1866,8 +1866,8 @@ private struct SpecialDirs {
 	 *
 	 * Versions of dub prior to v1.31.0 used to store  artifact under the
 	 * project directory, but this led to issues with packages stored on
-	 * read-only filesystem / location, and lingering artifacts scattered
-	 * through the filesystem.
+	 * read-only file system / location, and lingering artifacts scattered
+	 * through the file system.
```

Comment:
Wiki article uses both: https://en.wikipedia.org/wiki/File_system

Reply:
I went with `file system` because we want consistency and my spell checker just preferred it that way ^^